driver-service/src/main/java/by/sergo/driverservice/service/exception/ExceptionMessageUtil.java

```diff
@@ -0,0 +1,45 @@
+package by.sergo.driverservice.service.exception;
+
+import lombok.experimental.UtilityClass;
+
+import java.util.HashMap;
+
+@UtilityClass
+public class ExceptionMessageUtil {
+
+    public static String getCanNotCreateCarMessage() {
+        return "Can't create new car, please check input parameters";
```

Comment:
Зачем возвращать константу через метод, вынести просто как константу
```suggestion
        public static final String CAN_NOT_CREATE_CAR_MESSAGE = "Can't create new car, please check input parameters";
```